source/core.lisp

```diff
@@ -110,8 +102,10 @@ For each instruction it writes the result back to it."
              (destroy ()
                :report "Destroy the existing socket."
                (uiop:delete-file-if-exists (electron-socket-path interface)))))
+         (uiop:chdir (asdf:system-relative-pathname :cl-electron "source"))
```

Comment:
I don't exactly like how this is done, but I can't think of a better way yet.

Reply:
Fixed in f889e35.